### PR TITLE
Count of digits in python

### PR DESCRIPTION
**Pull request by tk315**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description
I wrote a code for count of digits in python on article on neoalgo docs which has got merged .
[https://github.com/TesseractCoding/NeoAlgo-Docs/pull/168](https://github.com/TesseractCoding/NeoAlgo-Docs/pull/168)

I would like to add that code here

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by tk315** [comment]
@HarshCasper Please Review

File: Python/math/count_of_digits.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        n = n//10
+    return cnt
+
+#Taking Input
+
+n = int(input("Enter the number :"))
+
+#Printing Output
+
+print("The count of digits in the given number is: {} ".format(count_of_digits(n)))
```
Comment (HarshCasper) [inline Python/math/count_of_digits.py]: Enclose in a `if __name__ == '__main__'` block

File: Python/math/count_of_digits.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+SAMPLE I
+
+INPUT
+Enter the number: 8965
+
+OUTPUT
+The count of digits in the given number is: 4
+
+"""
```
Comment (HarshCasper) [inline Python/math/count_of_digits.py]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/math/count_of_digits.py (hunk 1 above)
Comment (tk315) [reply Python/math/count_of_digits.py]: done

File: Python/math/count_of_digits.py (hunk 2 above)
Comment (tk315) [reply Python/math/count_of_digits.py]: done

**Review by HarshCasper — APPROVED** [review]

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**